src/components/Navbar.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import {
+  Collapse,
+  Stack,
+  IconButton,
+  Button,
+  Link,
+  useDisclosure,
+  Flex,
+  Image,
+  LinkBox,
+  LinkOverlay,
+  Hide,
+  Text,
+} from '@chakra-ui/react';
+import { HamburgerIcon, CloseIcon } from '@chakra-ui/icons';
+import { BsFillPersonPlusFill } from 'react-icons/bs';
+import AddNewUserModal from './AddNewUserModal';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "./AddNewUserModal"